src/components/Input/createInput.test.js

```diff
@@ -0,0 +1,13 @@
+import createInput from './createInput';
+
+describe('test for Input component', () => {
+  it('should return an <input /> element', () => {
+    const inputElement = createInput();
+    expect(inputElement.tagName).toBe('INPUT');
+  });
+
+  it('should be like the snaptshot', () => {
```

Comment:
It would be nice to have more complete description, in this way, if I read the test description I won't need to go to the snapshot to see what was tested. Try to answer the questions: What specifically am I testing here? What properties?

Reply:
Yes, It consider layout and class name as the properties to test.